### PR TITLE
Automated Test - COMMERCE-5857 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
https://issues.liferay.com/browse/COMMERCE-5857, cc: @lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path [hunk 1]
```diff
@@ -504,6 +504,11 @@
 	<td>//h4[contains(.,'Images')]//..//button[contains(@class,'btn-primary')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSERT_IMAGE_IS_SET</td>
+	<td>//span[contains(text(),'${key_itemName}')]/../../../../../../..//div[contains(@class,'product-card')]//div[contains(@style,'${key_itemImage}')]</td>
+	<td></td>
+</tr>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path:511]: Please rename this locator to `PRODUCT_CARD_CUSTOM_IMAGE`.
```suggestion
<tr>
	<td>PRODUCT_CARD_CUSTOM_IMAGE</td>
	<td>//span[contains(text(),'${key_itemName}')]/../../../../../../..//div[contains(@class,'product-card')]//div[contains(@style,'${key_itemImage}')]</td>
	<td></td>
</tr>
```

**Comment by lorenzolapescara** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 2]
```diff
@@ -118,12 +118,33 @@ definition {
...
+		CommerceProductImages.addProductImages(
+			imageTitles = "Blue",
+			validationImage = "true");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		AssertElementPresent(
+			key_itemImage = "Commerce_Blue",
+			key_itemName = "Blue T-Shirt",
+			locator1 = "CommerceEntry#ASSERT_IMAGE_IS_SET");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:147]: As for the comment above, please update `CommerceEntry#ASSERT_IMAGE_IS_SET` locator name.

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by AfonsoHSantos** [comment]
@lorenzolapescara fixed!

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**